annotation_suite_wrapper.sh

```diff
@@ -20,74 +20,94 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+source ../annotation-tools/annotation_suite_functions.sh
+
 # verify python3 and java installations
-command -v python3 >/dev/null 2>&1 || { echo "python3 is required to run this program - aborting..." >&2; exit 1; }
-command -v java >/dev/null 2>&1 || { echo "java is required to run this program - aborting..." >&2; exit 1; }
+command -v python3 >/dev/null 2>&1 || {
+    echo "python3 is required to run this program - aborting..." >&2
+    exit 1
+}
+command -v java >/dev/null 2>&1 || {
+    echo "java is required to run this program - aborting..." >&2
+    exit 1
+}
 
 function usage {
     echo "annotation_suite_wrapper.sh"
     echo -e "\t-i | --input-directory               input data directory for processing mutation data files [REQUIRED]"
-    echo -e "\t-o | --output-directory              output directory to write processed and annotated mutation data files to [REQUIRED]"
+    echo -e "\t-o | --output-directory              output directory to write processed and annotated mutation data files [REQUIRED]"
+    echo -e "\t-o | --error-directory               error directory to write annotation error reports to [REQUIRED]"
```

Comment:
Nice addition.  This is just a nice-to-have comment, but wanted to get your thoughts on just having this shell script be written in python. (Not urgent)

Reply:
Yes I would agree with writing it in python (easier tests, integration with other scripts, etc). Although it depends on if in the near future we still plan to keep using this fork and sync with the gn team's version, or just request for updates to the gn team's version or add wrapper functions to add in our desired changes to their functions?